nvflare/app_common/aggregators/dxo_aggregator.py

```diff
@@ -126,6 +126,7 @@ def accept(self, dxo: DXO, contributor_name, contribution_round, fl_ctx: FLConte
             n_iter = 1.0
         float_n_iter = float(n_iter)
         aggregation_weight = self.aggregation_weights.get(contributor_name)
+        self.log_debug(fl_ctx, f"Aggregator weights: {aggregation_weight}")
```

Comment:
this would print a log of message in the debug mode if the model is big.

Reply:
Good point. I moved this information to an existing debug log message that is only called during the aggregate() call.